cypress/e2e/marc-authority/create-new-marc-bib-verify-focus-behavior.cy.js

```diff
@@ -0,0 +1,130 @@
+import { DevTeams, Permissions, TestTypes } from '../../support/dictionary';
+import TopMenu from '../../support/fragments/topMenu';
+import Users from '../../support/fragments/users/users';
+import InventoryInstances from '../../support/fragments/inventory/inventoryInstances';
+import InventoryInstance from '../../support/fragments/inventory/inventoryInstance';
+import QuickMarcEditor from '../../support/fragments/quickMarcEditor';
+import inventoryKeyboardShortcuts from '../../support/fragments/inventory/inventoryKeyboardShortcuts';
+import InventoryHotkeys from '../../support/fragments/inventory/inventoryHotkeys';
+
+const hotKeys = InventoryHotkeys.hotKeys;
+function moveCursorBetweenSubfieldsAndCheck(rowNumber) {
```

Comment:
From my point of view it's better to move this logic to InventoryKeyboardShortcuts module